### PR TITLE
vdf docs: fix broken links and expand backup section a bit

### PR DESCRIPTION
**Pull request by listx**

### CODE REVIEW

**Comment by listx** [comment]
I am not sure this actually fixes the broken links because I don't know how to preview the rendered Markdown.

/cc @thockin

File: k8s.gcr.io/Vanity-Domain-Flip.md [hunk 1]
```diff
@@ -94,7 +94,7 @@ In order for a user to push to `k8s-artifacts-prod`, they must:
 
 1. Ensure that they have a [subproject staging repo][staging-subproject] (e.g.,
    `gcr.io/k8s-staging-foo` for the `foo` subproject).
-2. Add the promotion metadata in the [manifests subdirectory][] in the k8s.io repo.
+2. Add the promotion metadata in the [manifests subdirectory][./manifests] in the k8s.io repo.
```
Comment (spiffxp) [inline k8s.gcr.io/Vanity-Domain-Flip.md:97]: nit: prefer linking to the absolute path, and directly embedded links need to be in parens
```suggestion
2. Add the promotion metadata in the [manifests subdirectory](/k8s.gcr.io/manifests) in the k8s.io repo.
```

File: k8s.gcr.io/Vanity-Domain-Flip.md [hunk 2]
```diff
@@ -176,7 +182,7 @@ gcloud \
     $(printf "resource.type=project logName=%s resource.labels.project_id=%s" cip-audit-log k8s-artifacts-prod)
 ```
 
-The configuration for deploying the prod Cloud Run instance is [here][../infra/gcp/deploy-cip-auditor.sh].
+The configuration for deploying the prod Cloud Run instance is [here][../infra/gcp/cip-auditor/deploy.sh].
```
Comment (spiffxp) [inline k8s.gcr.io/Vanity-Domain-Flip.md:185]: same nit here
```suggestion
The configuration for deploying the prod Cloud Run instance is [here](/infra/gcp/cip-auditor/deploy.sh).
```

**Comment by bartsmykla** [comment]
As I see @spiffxp's comments were addressed and it looks good to me I let myself to approve/lgtm the PR, and if there will be another changes we can followup

/lgtm
/approve

**Pull request merged**